server/index.js

```diff
@@ -37,7 +43,7 @@ if (require.main === module) {
 }
 
 // Init bPanel
-module.exports = config => {
+module.exports = (config_ = {}) => {
```

Comment:
NIT: maybe use a prefix `_` instead of suffix `_`, I think its more common to see it as a prefix